src/Nethermind/Nethermind.Core/BlockHeader.cs

```diff
@@ -60,7 +60,7 @@ public BlockHeader(Keccak parentHash, Keccak ommersHash, Address beneficiary, UI
         public long GasUsed { get; set; }
         public long GasLimit { get; set; }
         public UInt256 Timestamp { get; set; }
-        public DateTime TimestampDate => DateTimeOffset.FromUnixTimeSeconds((long) Timestamp).DateTime;
+        public DateTime TimestampDate => DateTimeOffset.FromUnixTimeSeconds((long) Timestamp).LocalDateTime;
```

Comment:
Why local and not UTC?

Reply:
This is later used only for logging purposes, all other logs are in local time. We can have it in local time here, or convert to local time on logging. But like I said currently logging is the only use case.